frontend/src/components/Login/Login.tsx

```diff
@@ -0,0 +1,70 @@
+import './Login.css';
+import { ReactComponent as InstagramIcon } from 'assets/icons/instagram-icon.svg';
+import { Input } from './Input';
+import { InputButton } from './InputButton';
+import { Link } from 'react-router-dom';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { logIn } from 'services/auth';
+import { FirebaseError } from 'firebase/app';
+
+export function Login() {
+  const navigate = useNavigate();
+  const [error, setError] = useState('');
+  const [user, setUser] = useState({
```

Comment:
Is `user` a sensible name for an object that contains the login credentials?

Consider using a more appropriate name.